installer/release/bootstrap-install-script.sh.template

```diff
@@ -16,10 +16,6 @@ print_error_message() {
   echo "$(tput setaf 1)$errorMessage$(tput sgr0)"
 }
 
-##### Install NFS StorageClass
```

Comment:
how can I generate a new installer script?

Reply:
Run `make all` from inside `installer/release`. I'm going to add more doc about how to use it in another PR. Another pending task is to automatically build a new script when we cut a release.